styles/base/form.scss

```diff
@@ -93,28 +80,25 @@ label {
     z-index: 2;
     float: left;
     width: 100%;
+    border-right: 0;
+    border-left: 0;
+
+    &:focus { 
+      box-shadow: 0 0 2px 1px rgba(82,181,229,1);
```

Comment:
- I don't think our inputs have shadows. Let me verify.
- You use `rgba` for the opacity (the a (as in alpha) in rgb**a**). Also, we recommend to use hex if possible.